.openshift-ci/get_latest_helm_chart_versions.py

```diff
@@ -84,6 +96,83 @@ def get_latest_helm_chart_version_for_specific_release(chart_name, release):
         remove_helm_repo()
 
 
+def get_supported_helm_chart_versions():
+    add_helm_repo()
+    try:
+        update_helm_repo()
+        return __get_supported_helm_chart_versions()
+    finally:
+        remove_helm_repo()
+
+
+def __get_supported_helm_chart_versions():
+    supported_central_versions = []
+    supported_sensor_versions = []
+
+    supported_releases = get_supported_releases()
+    for release in supported_releases:
+        if __does_chart_exist("stackrox-central-services", release):
+            supported_central_versions.append(__get_latest_helm_chart_version_for_specific_release(
+                "stackrox-central-services", release)
+            )
+        else:
+            logging.debug(f"Supported version \"{release.major}.{release.minor}\" has no corresponding helm chart for "
+                          f"stackrox-central-services.")
+        if __does_chart_exist("stackrox-secured-cluster-services", release):
+            supported_sensor_versions.append(__get_latest_helm_chart_version_for_specific_release(
+                "stackrox-secured-cluster-services", release)
+            )
+        else:
+            logging.debug(f"Supported version \"{release.major}.{release.minor}\" has no corresponding helm chart for "
+                          f"stackrox-secured-cluster-services.")
+    return supported_central_versions, supported_sensor_versions
+
+
+def get_supported_releases():
+    supported_releases = []
+    data = __get_data_from_product_lifecycles_api()
+    releases = []
+    try:
+        releases = data["data"][0]["versions"]
+    except Exception as exception:
+        logging.debug(f"Found no RHACS releases in PRODUCT_LIFECYCLES_API at "
+                      f"{PRODUCT_LIFECYCLES_API}\n{repr(exception)}")
+    for release in releases:
+        if release["type"] != "End of life":
+            supported_releases.append(parse_release(release["name"]))
+    return supported_releases
+
+
+def __get_data_from_product_lifecycles_api():
+    req = Request(
+        url=PRODUCT_LIFECYCLES_API,
+        headers={'User-Agent': 'Mozilla/5.0'}
+    )
+    try:
+        response = urlopen(req)
+        with response:
+            try:
+                response_bytes = response.read()
+                response_string = response_bytes.decode('utf-8')
+                data = json.loads(response_string)
+                return data
+            except Exception as exception:
```

Comment:
`JSONDecodeError`. Not sure if can raise other exceptions though

Reply:
I don't think it can from what I just googled 👍